### DISCUSSION TITLE
What's the advantage of running the docker image over standalone executable?

### DISCUSSION THREAD

**Discussion by github-account1111** [go-shiori/shiori#419] [Q&A]
This wasn't obvious to me from the docs.
I use Docker for Python along with pip-tools to help dependency management, but I'm not sure how it translates to using a standalone program.
What would I be gaining?

**Comment by fmartingr** [comment]
I'm not sure on the particular scenario, but Container Images are provided here to easily test/deploy Shiori over multiple environments. Since containers became somehow the  _ new standard_ in the OPs side this is the simpler way for anyone with a working podman/docker environment to just run shiori and play with it, or to make a complex compose/ansible/helm/... to deploy under the infrastrucutre of their choice.

Of course, you could just download and execute the binary (and we also provide that), but again, I'm not sure the % of people that still does service deployment without containers or virtualization in any way.

Not sure this answers your original question!

**Comment by amitjethani** [comment]
I can add to the answer above: docker containers are incredibly popular with folks who run their own home labs, be it a home server (linux/TrueNAS), or a retail NAS product (Synology or QNAP), or just basic raspberry pi's with docker support. This allows for incredibly easy deployment and administration (including auto updates) for self-deployed containers. Take a peek at reddit's `r/selfhosted` to get a feel for just how large the community is!
